src/lib/Detabinator.js

```diff
@@ -0,0 +1,73 @@
+/**
```

Comment:
When moving this into here, we also have to [add those lines](https://github.com/demos-europe/demosplan-js-utils/blob/1861c1228f9672618a91f35e10c9253c7d5d2c13/LICENSE#L23-L47) to the demosplan-ui LICENSE file.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/71/commits/230e1d18d629101ef043c5263159e248e94391cb